spec/models/payment_reminder_config_spec.rb

```diff
@@ -16,6 +16,18 @@
 require "spec_helper"
 
 describe PaymentReminderConfig do
+  [:title, :text].each do |attr|
```

Comment:
Die specs greifen ein bisschen zu kurz. Da jeweils nur eines von beiden Attributen gesetzt wird ist das model zwangsläufig ungültig. Und ich den positiven fall (is valid wenn korrekt befüllt wird) sollte auch noch ergänzt werden